docker-storage-setup.sh

```diff
@@ -990,6 +989,10 @@ setup_docker_root_lv_fs() {
 }
 
 check_storage_options(){
+  if [ "$STORAGE_DRIVER" == "devicemapper" ] && [ -z "$CONTAINER_THINPOOL" ];then
+     Fatal "CONTAINER_THINPOOL must be defined for the devicemapper storage driver."
+  fi
+
```

Comment:
This code will not kick in if the user has not specified options to container-storage-setup.  In that code path docker-pool is always set.